src/app/features/course-info/course-info.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CourseInfoComponent } from './course-info.component';
+import { SharedModule } from '@app/shared/shared.module';
+
+const components = [CourseInfoComponent]
```

Comment:
Hmmm... is this really needed? How about adding `CourseInfoComponent` directly to the declarations and exports arrays  (since it is imported already)?

Reply:
I don't think it's needed, I just found the same pattern in the base repo's shared.module.ts. I thought this is something that is done regularly.
![image](https://github.com/user-attachments/assets/6418cb89-818d-4b98-a4de-237a95dfaef8)

Reply:
I see. I am not sure what was the intent here. Anyway, the update is there, thanks.